src/apps/allbridge/positions.ts

```diff
@@ -107,6 +107,21 @@ const hook: PositionsHook = {
               { address: tokenInfo.tokenAddress.toLowerCase(), networkId },
             ],
             availableShortcutIds: ['deposit', 'withdraw'],
+            shortcutTriggerArgs: {
+              deposit: {
+                token: {
+                  address: tokenInfo.tokenAddress.toLowerCase(),
+                  decimals: tokenInfo.decimals,
+                },
+                positionAddress: tokenInfo.poolAddress.toLowerCase(),
+              },
+              withdraw: {
+                token: {
+                  decimals: tokenInfo.decimals,
+                },
+                positionAddress: tokenInfo.poolAddress.toLowerCase(),
+              },
+            },
```

Comment:
I'm thinking maybe we use `shortcutTriggerArgs` to add only extra things the client doesn't already pass?

The client needs to be aware of the `deposit` and `withdraw` shortcut types and that it needs to pass token info an amounts.